keyboards/pk60/keymaps/default/keymap.c

```diff
@@ -37,6 +37,7 @@ void led_set_user(uint8_t usb_led) {
 	} else {
 		DDRF &= ~(1 << 4); PORTF &= ~(1 << 4);
 	}
+	led_set_user(usb_led);
```

Comment:
Pretty sure you don't want this there, unless you don't mind freezing your keyboard (function is calling itself, infinite loop)